packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -742,6 +742,12 @@ export class Combobox
         break;
       case "Delete":
       case "Backspace":
+        if (
+          this.selectionDisplay === "single" ||
```

Comment:
For readability, can you store this value in a local self-documenting variable and use here?